Sources/MessagingInApp/Views/UIKitInline.swift

```diff
@@ -113,66 +99,41 @@ public class InAppMessageView: UIView {
 
         heightConstraint?.constant = 0 // start at height 0 so the View does not show.
         getRootSuperview()?.layoutIfNeeded() // Since we modified constraint, perform a UI refresh to apply the change.
-
-        // Begin listening to the queue for new messages.
-        eventBus.addObserver(InAppMessagesFetchedEvent.self) { [weak self] _ in
-            // EventBus callback function might not be on UI thread.
-            // Switch to UI thread to update UI.
-            Task { @MainActor in
-                self?.refreshView()
-            }
-        }
     }
 
-    // Updates the state of the View, if needed. Call as often as you need if an event happens that may cause the View to need to update.
-    private func refreshView(forceShowNextMessage: Bool = false) {
-        defer {
-            // Always call the refreshViewListener at the end of the function to know processing is done.
-            refreshViewListener?()
-        }
-
-        guard let elementId = elementId else {
-            return // we cannot check if a message is available until element id set on View.
-        }
-
-        let queueOfMessagesForGivenElementId = localMessageQueue.getInlineMessages(forElementId: elementId)
-        let messageAvailableToDisplay = queueOfMessagesForGivenElementId.first { !hasBeenPreviouslyShown($0) }
-
-        if !forceShowNextMessage, isRenderingOrDisplayingAMessage {
-            // We are already displaying or rendering a messsage. Do not show another message until the current message is closed.
-            // The main reason for this is when a message is tracked as "opened", the Gist backend will not return this message on the next fetch call.
-            // We want to coninue showing a message even if the fetch no longer returns the message and the message is currently visible.
+    public func onMessageRendered(width: CGFloat, height: CGFloat) {
```

Comment:
All of the logic inside of these `delegate` functions is not new, but moved from different functions. 

Example: `onMessageRendered` function body is code that was in `onSizeChanged()`.